### ISSUE TITLE
March 2023 meeting

### ISSUE THREAD

**Issue by jar398** [ocapn/ocapn#36]
**Date:** Friday 24th of March 8pm UTC, 9pm UTC (convert to your timezone: https://www.timeanddate.com/worldclock/converter.html?iso=20230223T200000&p1=1440)
**Location:** https://bbb.sfconservancy.org/b/chr-pm5-cl8-ctl
**Scribe:** volunteer sought
**Agenda:** TBD
**Preparation:**
 - [ ] Update next meeting info at [ocapn.org](https://ocapn.org/)
 - [ ] Jonathan to review what's going on - get up to speed as chair
 - [ ] Agenda - I will update this issue description as items are added (easier to find than digging through issue comments)
        - Attempt resolution or a way forward on [https://github.com/ocapn/ocapn/issues/29](https://github.com/ocapn/ocapn/issues/29)

**Comment by ocdtrekkie** [comment]
Can you update the first URL to https://www.timeanddate.com/worldclock/converter.html?iso=20230324T200000&p1=1440 (the time of the next meeting, as opposed to the time of the previous one)?

**Comment by gibson042** [comment]
This runs afoul of a USA DST transition that creates conflict with some standing meetings... could it be an hour earlier, at 19:00 UTC?

**Comment by jar398** [comment]
@ocdtrekkie Thanks for the correction. Fixed now.

**Comment by zenhack** [comment]
I'm fine with doing 19:00 UTC to keep the USA local time the same; does this *not* work for anyone?

**Comment by tsyesika** [comment]
That time works for me too, I've pushed the time Friday the 24th of March 2023 between 19:00 (07:00 pm) and 20:00 (08:00 pm) UTC to the website. I think that should be at the same time as the last meeting for US folks. Let me know if that ought to be changed, it's not the time listed in your first [comment](https://github.com/ocapn/ocapn/issues/36#issue-1601583833) @jar398

**Comment by jar398** [comment]
I was waiting for this to settle out; I assume silence is assent but can Agoric (@kriskowal or @erights) please confirm 19:00 UTC (daylight time in US)?  Then I'll amend comment at top.  Thanks

**Comment by dckc** [comment]
can do 19:00Z Fri 24 Mar (2pm Chicago).

I have a conflict with 20:00Z.

**Comment by erights** [comment]
19:00 looks good. Most from Agoric will indeed need to drop promptly at 20:00.

**Comment by dckc** [comment]
re a way forward on #29, my preference is that we postpone it until we find out why we want/need it.

But since "Don't think about elephants!" just causes folks to think about elephants...

What I'd rather do instead is get back to [having folks assign themselves to what they want to work on](https://github.com/ocapn/ocapn/issues/28#issuecomment-1407469846).

**Comment by jar398** [comment]
I traveled back in time and updated the issue description (first comment).  I hope I got the timeanddate URL correct.

Dan, I think, has good advice as usual. I've found discussion so far to be useful because it suggests that maybe using URI syntax might not have any good use cases, and it surfaced the anatomy of ocapn ocaps. But I don't think anything important hinges on this question.

I would like to talk about purpose and process a little bit, but I know how that can get bogged down, so I plan to time-bound it since we can function (maybe not as well) without it. I will review the issues to come up with topics for the agenda that would be appropriate at this time, but if any of you have suggestions please speak up.

**Comment by tgrecojs** [comment]
Planning on joining in this months meeting as well. Thought I should give a heads up before popping in unannounced. 🙂

**Comment by jar398** [comment]
I didn't get a lot of response to my call for things to discuss.  I'll use one more day for brainstorming before I propose an agenda to you all.  I'm sure Jessica and/or I could fill the hour productively, but it would nice to hear from you all whether there are any issues or topics that could benefit from some attention at the meeting.  What is urgent?  What is nebulous?  What is contentious?

**Comment by dckc** [comment]
I suppose in the meanwhile, folks could make "I want to with on this" comments. Then we could sort the issues by recently updated as fodder for the agenda.

**Comment by jar398** [comment]
Right.  There's the 'triage' role which you have that would grant the needed authority. I don't know how others feel about giving a 'managing issues' authority to everyone though, or if there are any risks in doing so; I will have to look into this. In the meantime, as a stopgap, I am happy to do admin requests like changing issue properties, for those who don't yet have the required access.

**Comment by jar398** [comment]
@dckc The interop project is the topic that to me seems most exciting right now.  If you would spend 10 minutes before Friday putting some thoughts together I'm sure that would be enough to kick off a lively and perhaps even helpful discussion.  What do you think? We've already got two other items so it wouldn't have to fill the whole hour.

**Comment by cwebber** [comment]
I should be able to scribe

**Comment by tgrecojs** [comment]
is this the right link?

https://bbb.sfconservancy.org/html5client/join?sessionToken=euz4ek7nml4c2a4s

**Comment by tgrecojs** [comment]
@jar398 thanks! i was early an hour (thanks @dckc for informing me)

**Comment by dckc** [comment]
@tsyesika just noted

 - https://github.com/ocapn/syrup/pull/9
